package-lock.json

```diff
@@ -1,5 +1,45 @@
 {
     "name": "atlas-ui-framework",
-    "version": "2.5.3",
-    "lockfileVersion": 1
+    "version": "2.7.2",
+    "lockfileVersion": 1,
+    "requires": true,
+    "dependencies": {
+        "@types/prop-types": {
+            "version": "15.7.3",
+            "resolved": "https://registry.npmjs.org/@types/prop-types/-/prop-types-15.7.3.tgz",
+            "integrity": "sha512-KfRL3PuHmqQLOG+2tGpRO26Ctg+Cq1E01D2DMriKEATHgWLfeNDmq9e29Q9WIky0dQ3NPkd1mzYH8Lm936Z9qw==",
+            "dev": true
+        },
+        "@types/react": {
+            "version": "16.9.50",
```

Comment:
Aren't we on React 17 now for native with MX 9?

Reply:
We are, however the changes in `package-lock` are from running `npm i` and committing the file. No changes to `package.json` were made. I'd like to not touch the dependencies for now.

Reply:
The reason being is that this is a whole other tuna can - Atlas 2 and run on MX 8 + 9

Reply:
"Can of fish" - get it ;)